docs/source/release/v4.3.0/sans.rst

```diff
@@ -19,4 +19,9 @@ Improved
   can copy the history of a workspace to their clipboard or a file and the data
   will be reproduced on that machine without requiring editing of the script.
 
+Fixed
+#####
+- A zero monitor shift did not previously shift to account for the position
```

Comment:
Can you be a bit more specific in the notes (which algorithm ? which instruments? in which context?)